packages/suite/src/views/wallet/trading/common/TradingSelectedOffer/TradingOfferExchange/TradingOfferExchangeSignData.tsx

```diff
@@ -0,0 +1,48 @@
+import type { TradingExchangeType } from '@suite-common/trading';
+import { Button, Column, Divider, InfoItem } from '@trezor/components';
+import { spacings } from '@trezor/theme';
+
+import { AccountLabeling, Address, Translation } from 'src/components/suite';
+import { useTradingFormContext } from 'src/hooks/wallet/trading/form/useTradingCommonForm';
+
+export const TradingOfferExchangeSignData = () => {
+    const { device, account, callInProgress, selectedQuote, exchangeInfo, signDataAndConfirm } =
+        useTradingFormContext<TradingExchangeType>();
+
+    if (!selectedQuote) return null;
+
+    const { exchange, dexTx, receive, send } = selectedQuote;
+    if (!exchange || !dexTx || !receive || !send) return null;
+
+    const providerName =
+        exchangeInfo?.providerInfos[exchange]?.companyName || selectedQuote.exchange;
+
+    return (
```

Comment:
I like how clean this screen is.
However, a lot of users will see for the first time signing on the device.
I would love to have some banner here.
- sign data
- no tx fees
- swap can result into partial fill
- `Sign on Trezor and send`
- maybe we can use some extract from here https://help.1inch.io/en/articles/6800254-1inch-fusion-faq#h_3aa8adb5de
- 
![Screenshot 2025-02-28 at 15 04 20](https://github.com/user-attachments/assets/befb7a0b-e755-4722-9292-121530b23e61)

In case we do the above, I find this ok
![Screenshot 2025-02-28 at 15 07 13](https://github.com/user-attachments/assets/de1ec3d4-8fd2-4b32-b632-24b7f4961c1e)